ANSWERS.md

```diff
@@ -1,6 +1,15 @@
 ### Questions:
 1. What is the DOM?
+- Stands for Document Object Model, it's an  Application Programming interface between HTML and Javascripts. using javascript we can  dynamically access and update the content, structure, and style of a document.
+
 2. What is an event?
+- It is an action that occurs such as keystrokes and mouse movements ininiate by the user.
+
 3. What is an event listener?
+- An event listener is a function  that waits for an event to occur and then executes a callback.
+
 4. Why would we convert a NodeList into an Array?
+-  NodeList looks like an A misses methods like forEach, filter or map (which are part of the Array), so that why we convert a NodeList into an Array to inherit some buit-in Array's methods 
+
 5. What is a component? 
+- The idea behind components is to make reusable pieces of code and  without components , our applications become monolithic spaghetti monsters and nightmares.
```

Comment:
Lol, that is a very nice description of code made without components. I like your answers. They are clear and show that you know what you're talking about.